internal/server/web_server.go

```diff
@@ -116,6 +119,13 @@ func (h *RPCHandler) ServeHTTP(writer http.ResponseWriter, req *http.Request) {
 	zap.L().Debug("Request successfully routed.", zap.Any("requestBody", body))
 }
 
+func getClientId(req *http.Request) string {
+	// Try to get the id of the `client` via query parameter. Admittedly this is a little hacky, but it won't break
+	// functionality as query parameters aren't used in JSON RPC.
+	// A better solution is to pass the client via an HTTP header.
+	return req.URL.Query().Get("client")
```

Comment:
It'll just be an empty string if the param isn't specified, right?

Reply:
yep!